crates/fluvio-spu-schema/src/server/stream_fetch.rs

```diff
@@ -9,8 +9,8 @@ use std::marker::PhantomData;
 use std::io::{self, Read};
 use std::borrow::Cow;
 
-use dataplane::core::{Encoder, Decoder};
-use dataplane::api::Request;
+use fluvio_protocol::{Encoder, Decoder};
```

Comment:
inconsistent usage here. either we export fluvio_protocol from data plane or we don't.   We should export it as before.